src/applications/mhv-landing-page/components/alerts/AlertMhvBasicAccount.jsx

```diff
@@ -2,29 +2,29 @@ import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 // eslint-disable-next-line import/no-named-default
 import { default as recordEventFn } from '~/platform/monitoring/record-event';
+import { logoutUrl } from 'platform/user/authentication/utilities';
 import { logoutUrlSiS } from '~/platform/utilities/oauth/utilities';
-import { VaAlert } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import {
```

Comment:
That's a good question. I assumed I should use the react-bindings for the VA web components, since it was provided to us, and we're writing React code.